main/getting-started/beta.md

```diff
@@ -74,16 +74,13 @@ has not been exercised by the deadline, its underlying assets are paid to the co
 as a refund of their escrowed assets.
 
 ## Developer Experience
-Agoric is a team of engineers. We've based our developer experience on our decades of open-source software experience. In our beta, you can expect the following:
+Agoric is a team of engineers. We've based our developer experience on our decades of open-source software experience to make [getting started](/getting-started/) as easy as possible. In our beta, you can expect the following:
```

Comment:
Dropping the separate paragraph below is a fine idea.

I think @tgrecojs's suggestion to have this link take them to the quick start directions instead  (https://docs.agoric.com/getting-started/before-using-agoric.html) is a good idea.

The one here was already linked on line 32 above, where the anchor for the link should be changed (as he suggested) to "Learn more about getting started with the Agoric SDK".